script/Ciel/Day 5/05_01_CIEL05_1_1.txt

```diff
@@ -82,7 +82,8 @@ The foul sound of him swallowing worms its way in through my ears and into my br
 [sha:99f6e134f6b2bfbf6d103cfca7e4dd26df9c67cc]{
 -- Page 276, Offset 11595.
 -- ―――目の前で食事をされるのはこれで二度目だ。
-―――This makes the second time I've seen someone treated as food before my eyes.
+// group pr check: alt suggestion: That makes it the second time
+―――This makes it the second time I've seen someone treated as food before my eyes.
```

Comment:
Not sure I agree with the addition of `it`. Non-it variant much more common on ngrams too.